src/app-service-registration.ts

```diff
@@ -62,20 +78,20 @@ export class AppServiceRegistration {
     private hsToken: string|null = null;
     private asToken: string|null = null;
     private senderLocalpart: string|null = null;
-    private rateLimited = true;
+    private rateLimited: boolean|undefined = undefined;
     /**
      * **Experimental**  
      * Signal to the homeserver that this appservice will accept ephemeral events.
      */
-    public pushEphemeral = false;
+    public pushEphemeral: boolean|undefined = undefined;
```

Comment:
Does `pushEphemeral` need a getter to default it to `false`?

Reply:
Should be good now. Can haz :heavy_check_mark: in the review UI?

Reply:
Done 😃